huntsman/tests/test_camera.py

```diff
@@ -233,7 +235,7 @@ def test_exposure_scaling(camera, tmpdir):
     else:
         fits_path = str(tmpdir.join('test_exposure_scaling.fits'))
         camera.take_exposure(filename=fits_path, dark=True, blocking=True)
-        image_data, image_header = fits.getdata(fits_path, header=True)
+        image_data, image_header = fits_utils.getdata(fits_path, header=True)
```

Comment:
This was an error. Not sure how test was passing.

Reply:
This tests usually gets skipped as it depends on a camera having a bit depth other than 16 bits/pixel and the huntsman-pocs test suite only tests with a single type of simulated camera.